ci/packages/build.go

```diff
@@ -76,10 +76,10 @@ func buildBinary(source, target string) error {
 	if !ok {
 		targetArch = runtime.GOARCH
 	}
-	return buildBinaryFor(source, target, targetOS, targetArch)
+	return buildBinaryFor(source, target, targetOS, targetArch, false)
```

Comment:
Just a thought that any further extensions of `buildBinaryFor` signatures could be done as `buildBinaryFor(..., opts... BuildOptions)` for better readability.